src/main_targil1.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import _ from 'lodash';
```

Comment:
Lodash is not a big library, but in general, it contains many methods that you don't actually need.
A better solution would be to import it like this:
```js
import range from 'lodash/range'
import random from 'lodash/random'
```